build/namespace.wl

```diff
@@ -1,23 +1,40 @@
 (* ::Package:: *)
 
-namespace = Select[Names["*"], PrintableASCIIQ];
+Begin["wl`"];
 
 
-Monitor[
-	Quiet[
-		usages = Table[
-			namespace[[i]] -> (ToExpression[namespace[[i]] <> "::usage"]),
-		{i, Length @ namespace}],
-	Message::name],
-	ProgressIndicator[i / Length@namespace]
+namespace = Select[Names["System`*"], PrintableASCIIQ];
+namespaceSize = Length[namespace];
+
+
+Monitor[Quiet[
+	usages = Table[
+		namespace[[i]] -> ToExpression[namespace[[i]] <> "::usage"],
+		{i, namespaceSize}
+	], Message::name],
+	ProgressIndicator[i / namespaceSize]
 ];
 
 
-Export[NotebookDirectory[] <> "usage-dict.json",
-	Select[usages, Head[Values[#]] === String &]
+usageDictionary = Select[usages, Head[Values[#]] === String &];
+usageAbsentSymbols = Keys @ Select[usages, Head[Values[#]] =!= String &];
+
+
+getAtomic[expr_, orders_List] := If[Length @ orders == 0,
+	expr,
+	getAtomic[Level[expr, 1][[orders[[1]]]], Rest @ orders]
 ];
 
 
-Export[NotebookDirectory[] <> "usage-absent.json", Keys[
-	Select[usages, Head[Values[#]] =!= String &]
-]];
+namedCharacters = StringTake[getAtomic[#, {1, 1, -1, 1}], {3, -2}]& /@ Import[FileNameJoin[{
+	$InstallationDirectory,
+	"Documentation",
+	$Language,
+	"System/Guides/ListingOfNamedCharacters.nb"
+}], {"Cells", "GuideText"}];
+
+
+DumpSave[NotebookDirectory[] <> "wldata.mx", "wl`"];
```

Comment:
Should there be a notebook committed if this file expects one to be saved?

Reply:
No, the *wldata.mx* will only be used by *classify.wl*, not by sublime. Anyone who wants the file can generate it by himself. Given that the file is big and binary, a better practice is to leave it ignored.